index.js

```diff
@@ -10,11 +11,18 @@ const router = require('./app/router');
 const PORT = process.env.PORT || 4000
 
 
+const url = 'mongodb://localhost:27017'
+const dbName = 'monProjetMongo';
+
+MongoClient.connect(url, function(err, client) {
+  console.log("Connecté à MongoDB");
+});
+
 app.use(router);
 
 
-app.listen(PORT, () => {
-    console.log(`You can launched the server on http://localhost:${PORT}`)
-});
+// app.listen(PORT, () => {
+//    console.log(`You can launched the server on http://localhost:${PORT}`)
+// });
```

Comment:
```suggestion
app.listen(PORT, () => {
  console.log(`You can launched the server on http://localhost:${PORT}`)
});
```

Pourquoi est-ce que tu mets en commentaire ce bout de code ?